packages/client/src/components/Markdown/renderers/BlockRenderer.tsx

```diff
@@ -7,8 +7,7 @@ import { StyledParagraph } from "./BlockRendererStyles";
  * we want to render the visual without wrapping it in a Paragraph as it's invalid
  * to wrap a figure-tag with a p-tag.
  */
-export default (props) => {
-  const { children } = props;
+export default ({ children }: { children: any }) => {
```

Comment:
Can you replace any with React.ReactChildren?

Reply:
Nah, this fails: `JSDoc types can only be used inside documentation comments.ts(8020)`

Reply:
Fixed